<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupEdge</class>
 <widget class="QDialog" name="PopupEdge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edge properties</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* http://paletton.com/#uid=13U0u0kd1oO0hT21i-bwUkbDIde
 * color 1: #E9EAEB
 * color 2: #B3B4B9
 * color 3: #585E88
 * color 4: #0E1B6E
 * color 5: #071048
 */

QDialog {
  background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E9EAEB, stop: 1 #B3B4B9);
}

#leftNode, #rightNode {
  border-radius: 100;
  border: 5px solid #071048;
  background: #585E88;
}

#labelTextbox {
  background-color: #ecf0f1
}

#lineLeft, #lineRight {
  background: #071048;
}

#leftArrow {
	border-top: 20px solid QColor(0,0,0,0);
	border-right: 40px solid red;
	border-bottom: 20px solid QColor(0,0,0,0);
}

#rightArrow {
	border-top: 20px solid QColor(0,0,0,0);
	border-left: 40px solid green;
	border-bottom: 20px solid QColor(0,0,0,0);
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="leftNode">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>80</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="rightNode">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="leftNodeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LeftNode</string>
   </property>
  </widget>
  <widget class="QLabel" name="rightNodeLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>RightNode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="lineRight">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>340</width>
     <height>6</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="lineLeft">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>340</width>
     <height>6</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="leftArrow">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>214</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="rightArrow">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>103</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftToRightWeight">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>only numbers are accepted</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightToLeftWeight">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>only numbers are accepted</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rightToLeftCheckbox">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>224</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="leftToRightCheckbox">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>114</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteEdge">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>364</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PopupEdge</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PopupEdge</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteEdge</sender>
   <signal>clicked()</signal>
   <receiver>PopupEdge</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
